test/integration/features/basic.feature

```diff
@@ -10,6 +10,61 @@ Feature: Basic
      Then Minishift should have state "Running"
       And Minishift should have a valid IP address
 
+  Scenario: User is able to do ssh into Minishift VM
+    Given Minishift has state "Running"
+     When executing "minishift ssh echo hello"
+     Then stderr should be empty
+      And exitcode should equal 0
+      And stdout should contain
+      """
+      hello
+      """
+
+  Scenario: User is able to set custom Docker specific environment variables
+    Given Minishift has state "Running"
+     When executing "minishift ssh cat /var/lib/boot2docker/profile"
+     Then stderr should be empty
+      And exitcode should equal 0
+      And stdout should contain
+      """
+      export "FOO=BAR"
+      export "BAZ=BAT"
+      """
+
+  Scenario Outline: User can set, get, view and unset values in configuration file
+    User is able to setup persistent configuration of Minishift using "config" command
+    and its subcommands.
+    Given Minishift has state "Running"
+     When executing "minishift config set <property> <value>"
+     Then JSON config file contains key "<property>" with value "<value>"
+      And stdout of command "minishift config get <property>" is equal to "<value>"
+      And stdout of command "minishift config view --format {{.ConfigKey}}:{{.ConfigValue}}" contains "<property>:<value>"
+     When executing "minishift config unset <property>"
+     Then stdout of command "minishift config get <property>" is equal to "<nil>"
+      And JSON config file does not contain key "<property>"
+      
+  Examples: Config values to work with
+    | property  | value |
+    | disk-size | 22g   |
+    | memory    | 2222  |
+    | cpus      | 3     |
+
+  Scenario: Commands console and ip returns valid ip
+    User is able to get ip of Minishift VM with command "ip" and url adress of OpenShift instance
+    by running command "console" with flag "--url". Both commands should contain same IP.
+    Given Minishift has state "Running"
+     When executing "minishift console --url"
+     Then stdout should be valid address
+      And "minishift ip" should return same ip
+
+  Scenario: OpenShift is ready after startup
+    After startup of Minishift OpenShift instance should respond correctly on its html endpoints
+    and OpenShift web console should be accessible.
+    Given Minishift has state "Running"
+     Then response of OpenShift at "/healthz" contains "ok"
```

Comment:
@hferentschik So I did some research today - it would be ideal to have this replacement of ${variable} for actual value accessible in every step definition. My idea here is to use nested steps which were introduced 5 days ago in godog. For example like this:

Step: 
`Then body of http request on "${server_ip}/health" is "ok"`
Matching:
```
s.Step(`step:"minishift ip" should return same ip`, m.shouldReturnSameIp)
s.Step(`(.*)`, injectVar)
```
injectVar step definition:
```
func injectVar(stepName string) golang.Steps {
   [supplementing ${server_ip} for actual IP of server in stepName string]
   return golang.Steps{
     fmt.Sprintf(`step:%s`, stepName),
   }
}
```
Doing it like this every step is going to trigger injectVar step implementation, which will inject the variable value and then will call actual step implementation. I am having problems with fact that godog.Steps seems to be undefined whatever I try to do, but theoretically is this a way to go?

Second option is to implement the replacement of ${var} only specified steps.

Did not found any other option yet...